music_theory_api/settings.py

```diff
@@ -72,7 +73,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [str(BASE_DIR.joinpath('templates'))],
```

Comment:
I don't believe str is needed here.  Can you confirm?

Reply:
You are correct! It has been removed :)